src/main/java/edu/cornell/kfs/sys/service/impl/ErrorMessageUtilsServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package edu.cornell.kfs.sys.service.impl;
+
+import java.text.MessageFormat;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.rice.core.api.config.property.ConfigurationService;
+
+import edu.cornell.kfs.sys.service.ErrorMessageUtilsService;
+
+public class ErrorMessageUtilsServiceImpl implements ErrorMessageUtilsService{
+    protected ConfigurationService configurationService;
+    
+    public String createErrorString(String errorKey, String... params) {
```

Comment:
I am not sure I understand the question. Is there a easier way to do this?

I have basically copied this from AttributeValidatingTypeServiceBase and extracted it in this utility service that can be used outside of KFS documents too.